src/java.base/share/classes/java/lang/String.java

```diff
@@ -3180,12 +3180,12 @@ public String translateEscapes() {
      * string. The function should expect a single String argument
      * and produce an {@code R} result.
      * <p>
-     * Any exception thrown by {@code f()} will be propagated to the
+     * Any exception thrown by {@code f.apply()} will be propagated to the
      * caller.
      *
-     * @param f    functional interface to a apply
+     * @param f    a function to apply
```

Comment:
Shouldn't this be "_the_ function to apply"? Grep for other instance of "function to apply" in javadocs :)

Reply:
I had grepped that before pushing and the results were inconclusive. For example, compare the below fragments from the same file:

```
    /**
...
     * @param mapper the mapping function to apply to each element
...
     */
    public static <T>
    Collector<T, ?, LongSummaryStatistics> summarizingLong(ToLongFunction<? super T> mapper)
```
```
    /**
...
     * @param mapper a mapping function to apply to each element
...
     */
    public static <T>
    Collector<T, ?, DoubleSummaryStatistics> summarizingDouble(ToDoubleFunction<? super T> mapper)
```

Reply:
Hm! Better to defer to native speakers then...